docs/api-docs/partner/technology-program

```diff
@@ -0,0 +1,132 @@
+
+See the Technology Partner Program application/requirements in the section below to find out how to apply.
+
+## Technology Partner Program application/requirements
+Only official partners can submit an app to the BigCommerce App Marketplace, also known as the App store. BigCommerce offers app developers free sandbox stores through its Technology Partner Program. To be approved as a partner, you will need:
+
+* A website showcasing your agency or technology solution
+* Intent to offer your app publicly
+* Ability to support your app's users
+* Evidence of expertise in app development
+
+Also, there is a list of general requirements in [App Store Approval Requirements](https://developer.bigcommerce.com/api-docs/partner/app-store-approval-requirements), which provides best practices for you to follow.
+
+How to apply:
+1. Go to the [Partner Portal](https://partners.bigcommerce.com/).
+   
+2. Click **Apply Today**.
+   
+![Partner Apply](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/becoming-a-partner-01.png "Partner Apply")
+
+3. Enter your company email address.
+   
+4. Click the **Technology Partner Program** button.
+   
+![Partner Registration](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/becoming-a-partner-02.png "Partner Registration")
+
+5. Complete the member registration.
+6. Click the **Complete Registration** button. 
+7. Once your application is approved, check your email for your Partner ID.
+
+
+Once approved, you will receive one or more emails listing your partner credentials and next steps. Your **Partner ID** will be required to submit your app for App Marketplace consideration. If you do not receive these credentials by email, please contact [appstore@bigcommerce.com](mailto:appstore@bigcommerce.com).
+
+## App Marketplace Overview
+To build a Marketplace app, you will need:
+
+* A sandbox store
+* A developer account on the [BigCommerce Developer Portal](https://devtools.bigcommerce.com/my/apps)
+* BigCommerce partner credentials 
+
+We encourage you to create your sandbox store and your Dev Tools account using the same login credentials. A sandbox store cannot process transactions and is for developing and testing apps without the 15-day time limit of a trial store. 
+
+> **Note**
+> Your Developer Portal account is an entirely separate account outside of the Partner Portal.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--warning">
+<div class="HubBlock-content">
+    
+<!-- theme: warning -->
+
+### Trials erased after 15 days
+> Although you do not need a sandbox store for app development, it is best practice. If you opt to develop on a trial store, keep in mind the store and data will be erased after 15 days, and you will have to create a new trial store to proceed.
+
+
+</div>
+</div>
+</div>
+
+### Creating a sandbox store
```

Comment:
I think this section should be removed from this article and included in the BigCommerce API Basics -> Getting started.
```suggestion
### Creating a sandbox store
```